components/controls/RadioCard.css

```diff
@@ -66,6 +71,15 @@
   border-color: var(--darkBlue);
 }
 
+.radio-card-vertical:not(.disabled):hover {
+  box-shadow: none;
+  transform: none;
```

Comment:
It could be good to have comment explaining why we don't want to have the transform in the vertical form